research/conditional/utils/conditional_trainer.py

```diff
@@ -135,13 +138,13 @@ def train(self, n_steps: int):
             if step % 1000 == 0:
                 print(f"Step {step}")
 
-            if (
-                self.model_type == "gpt"
-                and self.decoding_logging_steps > 0
-                and step % self.decoding_logging_steps == 0
-                and self.is_process_logging
-            ):
-                self._decode_samples(step)
+            # if (
```

Comment:
Yo, the desired solution I believe is to set decoding_logging_steps to 0 to disable decoding.